### ISSUE TITLE
Refactor `Cli::into_str_args`

### ISSUE THREAD

**Issue by itegulov** [near/mpc-recovery#356]
### Description

This is a very manual and error-prone pattern. We could invest some time into creating a proper trait and a derive macro so that it works on any clap struct hierarchy. Honestly, I am surprised there is nothing ready in clap itself

CC @ChaoticTempest

**Comment by ChaoticTempest** [comment]
Yeah, seems like with the amount of times we're actually doing this pattern, it's probably worth creating a library for it
